Tests/XcodeGenKitTests/ProjectSpecTests.swift

```diff
@@ -85,7 +86,8 @@ func projectSpecTests() {
 
                 try expectValidationError(spec, .missingConfigTypeForGeneratedTargetScheme(target: "target1", configType: .debug))
                 try expectValidationError(spec, .missingConfigTypeForGeneratedTargetScheme(target: "target1", configType: .release))
-
```

Comment:
I think it'd be great to also test the other error that you throw when the file doesn't exist. You could do that by injecting a function to the `SpecValidation` with the signature `exists: (Path) -> Bool = { $0.exists}` and updating your code to be:

```swift
if !exists(path) {
  errors.append(.invalidHeaderPathForTarget(target: target.name, path: header))
}
```
Then you could inject that function in the test returning `false`.